src/Components/creatorPage/OtherWorks.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import propTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './OtherWorks.css';
+// import { v4 as uuidv4 } from 'uuid';
+
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+// Authored and illustrated work components
+function CreatedWorksCard({ authorId }) {
+  const [imageTitleObject, setImageTitleObject] = useState([]);
```

Comment:
the other object is named after the type of work: "illustratedObject". Can you rename this one to authoredObject to keep consistent between the two? Better yet, maybe rename them to "authoredWorks" and "illustratedWorks" respectively so the names are more connotative.